### PR TITLE
faccess fixture

### PR DESCRIPTION
**Pull request by abuts**
simple solution of #1022 using fopen operation which defines char as 8-bits symbols

### CODE REVIEW

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: horace_core/algorithms/@cut_data_from_file_job/private/put_sqw_data_npix_and_pix_to_file_.m [hunk 1]
```diff
@@ -41,7 +41,7 @@
     if exist(outfile,'file') == 2
         delete(outfile);
     end
-    fid=fopen(outfile,'A');    % no automatic flushing: can be faster
+    fid=fopen(outfile,'A','l','Windows-1252');    % no automatic flushing: can be faster
```
[the remark this answers is not in the document]
Comment (oerc0122) [reply horace_core/algorithms/@cut_data_from_file_job/private/put_sqw_data_npix_and_pix_to_file_.m:44]: As mentioned before, I agree with @cmarooney-stfc . If we want to read N bytes, we should use the appropriate length types to read them rather than setting an encoding. Any of the following (https://uk.mathworks.com/help/matlab/ref/fread.html#btp1twt-1-precision):
```
uint8
int8
integer*1
uchar
unsigned char
ubit8
schar
signed char
bit8
char*1
```
Would work. It should match the type you're actually trying to read to make it clear, I would recommend `integer*1` since this is the appropriate MATLAB type.

[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/algorithms/@cut_data_from_file_job/private/put_sqw_data_npix_and_pix_to_file_.m:44]: it is fully portable. The suggestion applies to read, and I applied the formatting to open, making `char == char*1`. There are subtle differences in parsing read data for different types of input and I do not want suddently hit these subtletly

File: horace_core/sqw/file_io/@horace_binfile_interface/private/read_header_.m [hunk 2]
```diff
@@ -43,7 +43,7 @@
     fid=file_name;  % copy fid
     do_fseek(fid,0,'bof');  % set the file position indicator to the start of the file
 else
-    fid=fopen(file_name,permission_req);
+    fid=fopen(file_name,permission_req,'l');
```
Comment (oerc0122) [inline horace_core/sqw/file_io/@horace_binfile_interface/private/read_header_.m:46]: Why no encoding?

File: horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m [hunk 3]
```diff
@@ -174,79 +174,11 @@
             % 8: 3xNpix array
```
Comment (oerc0122) [inline horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: Shouldn't be included in this PR

Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: That's the conflict resolution (and subtle bug in init)

Comment (oerc0122) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: It still shouldn't be included in this PR

Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: there are no other way of resolving conflicts

Comment (oerc0122) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]:
```
git checkout master horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m
git commit --amend
git push -f
```

Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: So, I would be pushing to master invalid (outdated) code? I do not want this -- I will certainly forget to change it to recent version

Comment (oerc0122) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: Where are you pushing outdated code to master? What are you talking about? This pulls the original file from master and puts it in this branch. If you need to make the change of `sqw_fopen` to the file, then you can do so before you commit. If you want to keep the other changes then you can check this filee out to another branch and open that as a PR.

Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: I have no idea what the problem is, but I've seen the conflict in PixelDataFileBacked and resolved it with the latest changes to code.

Comment (oerc0122) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: Because it's a significant change not relevant to this PR

Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m:174]: If I see the issue, I am fixing it whatever possible and convenient. The relevance is very relative thing.
